### PR TITLE
chore(elbv2): Improve verification of "elbv2.ListenerCondition.pathPatterns" 

### LINKED ISSUE #10065: [elbv2] A validate feature for "elbv2.ListenerCondition.pathPatterns" for "A rule can only have '5' condition values"
```typescript
elbv2.ListenerCondition.pathPatterns(['/r1', '/r2', '/r3', '/r4', '/r5','/r6']), 
```
elbv2.ListenerCondition.pathPatterns length > 5 will throw "A rule can only have '5' condition values" exception after "creating CloudFormation changeset..." and stack will roolback.

### Proposed Solution

If there is a validate before "creating CloudFormation changeset..." will detect problems earlier and save time.

* [x] :wave: I may be able to implement this feature request
* [ ] :warning: This feature might incur a breaking change

---

This is a :rocket: Feature Request

### PR DESCRIPTION
**Pull request by jialechan**
fixes #10065

----

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

### CODE REVIEW

File: packages/@aws-cdk/aws-elasticloadbalancingv2/test/alb/conditions.test.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import '@aws-cdk/assert/jest';
+import * as elbv2 from '../../lib';
+
+describe('tests', () => {
+  test('pathPatterns length should be less than or equal to 5', () => {
```
Comment (njlynch) [inline packages/@aws-cdk/aws-elasticloadbalancingv2/test/alb/conditions.test.ts:5]: Can you add a positive case as well?

**Review by njlynch — COMMENTED** [review]
Thanks for the PR! It looks like the build is complaining of some formatting issues, but otherwise looks good.

File: packages/@aws-cdk/aws-elasticloadbalancingv2/test/alb/conditions.test.ts (hunk 1 above)
Comment (njlynch) [reply packages/@aws-cdk/aws-elasticloadbalancingv2/test/alb/conditions.test.ts:5]: I see a commit labeled "add positive case", but don't see it in the diff yet. Did it get lost in a merge somewhere?

Comment (jialechan) [reply packages/@aws-cdk/aws-elasticloadbalancingv2/test/alb/conditions.test.ts]: The unit test has been updated, please check whether it meets the requirements, thanks

File: packages/@aws-cdk/aws-elasticloadbalancingv2/test/alb/conditions.test.ts [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+import * as elbv2 from '../../lib';
+
+describe('tests', () => {
+
+  test('pathPatterns length greater than 5 will throw exception', () => {
+    //GIVEN
+    const array = ['/u1', '/u2', '/u3', '/u4', '/u5'];
+
+    //WHEN
+    elbv2.ListenerCondition.pathPatterns(array);
```
Comment (njlynch) [inline packages/@aws-cdk/aws-elasticloadbalancingv2/test/alb/conditions.test.ts:11]:
```suggestion
    elbv2.ListenerCondition.pathPatterns(array); // Does not throw
```

**Review by njlynch — APPROVED** [review]

**Pull request merged**